index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rizwan hasan portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+  </head>
+
+  <body>
+
+    <nav>
+      <div class="navbar">
+        <h1 class="logo"> My Logo </h1>
+
+        <ul>
+          <li class="nav-items"><a href="#">Portfolio</a></li>
+          <li class="nav-items"><a href="#">About</a></li>
+          <li class="nav-items"><a href="#">Contact</a></li>
+        </ul>
+      </div>
+                  
+    </nav>
+
+    <section class="homepage">
+      <div class="intro">
+
+        <h1>I'm Rizwan <br>Glad to meet you!</h1>
+        <br>
+        <br>
+        <p>
+          I am a software developer. I have done some interesting projects using
+          react js, node js and ruby on rails. I am learning new things and
+          implementing them instantly. I can help you build a project, feature
+          or website look through some of work and experience if you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <p class="socials">Lets connect</p>
+        <div class="S-icons">
+          <ul>
+              <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+              <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+              <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+              <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
+          </ul>
+        </div>                
+      </div>
+    </section>
```

Comment:
Sadly, something is wrong with your css or html structure 😥

![image](https://user-images.githubusercontent.com/15807343/148621971-b3422e6d-0913-49b2-b06c-8aa9ed327ac5.png)

Please fix it so you have almost an identical design as figma's 📝

Reply:
overlay error has been resolved, by adding margins and using flexbox